db/upgrade.php

```diff
@@ -46,6 +46,7 @@ function xmldb_block_navbuttons_upgrade($oldversion) {
         $table->add_field('extra2link', XMLDB_TYPE_TEXT, 'medium', null, null, null, null);
         $table->add_field('extra2title', XMLDB_TYPE_TEXT, 'small', null, null, null, null);
         $table->add_field('extra2openin', XMLDB_TYPE_INTEGER, '4', null, XMLDB_NOTNULL, null, '1');
+        $table->add_field('completebuttonshow', XMLDB_TYPE_INTEGER, '4', null, XMLDB_NOTNULL, null, '1');
```

Comment:
Just realised that I'd missed this one earlier - this is useless as an upgrade.
If I upgrade from version 2016052000 to 2017031500, then this line will not result in the new field being added to the table.
Please remove this line from here and add a proper upgrade that will add this field if it doesn't already exist (the XMLDB editor built into Moodle will generate the correct code for you).